src/adapt/net.js

```diff
@@ -66,7 +66,7 @@ adapt.net.ajax = function(url, opt_type, opt_method, opt_data, opt_contentType)
                     response.responseXML = request.responseXML;
                     response.contentType = request.responseXML.contentType;
                 } else if ((!opt_type || opt_type === adapt.net.XMLHttpRequestResponseType.DOCUMENT) &&
-                    request.response &&
+                    request.response instanceof HTMLDocument &&
                     String(request.response) == '[object HTMLDocument]') {
```

Comment:
This last condition is no longer necessary since it is equivalent to the `instanceof` check.